src/Store/Store.ts

```diff
@@ -0,0 +1,51 @@
+import EventBus from "../services/EventBus";
+
+export type TState = Record<string, any>;
+
+export enum StoreEvents {
+	Updated = 'updated',
+}
+
+class Store extends EventBus {
+
+	static _instance: Store;
+	static STORE_NAME = 'myAppStore';
```

Comment:
Если это константа, давай пометим ее как readonly
```suggestion
	static readonly STORE_NAME = 'myAppStore';
```